<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExamEditor</class>
 <widget class="QDialog" name="ExamEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExamEditor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Modality</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="modality">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Device</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="device">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Region</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="region">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Contrast media</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="contrast">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="contrastAmount">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Cost</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="quantity">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cost">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ordered by</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="orderedBy">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Examination date</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="examDate">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>478</width>
       <height>57</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
